src/javascript/dimensions-metrics-explorer/cubes.tsx

```diff
@@ -0,0 +1,28 @@
+// In the future, this file will populate cubes from a devsite URL.
+// For now it's hardcoded.
+
+import cubes from "./ga_cubes.json";
+import { reduce } from "lodash";
+import { Set, Map } from "immutable";
+
+// Mapping of Column names to sets of cubes
+export type CubesByColumn = Map<string, Set<string>>;
+
+const buildCubes = (): CubesByColumn => {
+  return reduce(
+    cubes,
+    (cubesByColumn, columnList, cubeName) =>
+      reduce(
+        columnList,
+        (cubesByColumn, _, columnName) =>
+          cubesByColumn.update(columnName, Set(), cubes => cubes.add(cubeName)),
+        cubesByColumn
+      ),
+    Map()
+  );
+};
+
+// This is a promise because in the future it will be loaded via a fetch.
+export const cubesByColumn: Promise<CubesByColumn> = Promise.resolve(
```

Comment:
since this is in the bundle, I think this gets invoked regardless of page, it's probably better to wrap this is a callback so it's not invoked when not needed.

Reply:
I'll double check, but I'm pretty sure our build script produces one bundle per page, so this still only gets loaded when the DME page specifically is loaded. Still, I can look into wrapping it in a function with a `lodash.once`.